src/main/kotlin/racing/view/CarRacingInputView.kt

```diff
@@ -1,12 +1,21 @@
-package racing
+package racing.view
+
+import racing.dto.CarRacingInputData
 
 object CarRacingInputView {
     fun input(): CarRacingInputData {
         println("경주할 자동차 이름을 입력하세요(이름은 쉼표(,)를 기준으로 구분).")
         val carNames: List<String> = readLine()!!.split(",")
+
+        carNames.forEach(::checkCarNameLength)
+
         println("시도할 횟수는 몇 회인가요?")
         val moveCount = readLine()!!.toInt()
 
         return CarRacingInputData(carNames = carNames, moveCount = moveCount)
     }
+
+    fun checkCarNameLength(carName: String) {
+        check(carName.length <= 5) { "차 이름은 5글자를 초과 할 수 없습니다." }
```

Comment:
```suggestion
        check(carName.length <= 5) { "차 이름은 5글자 이하여야 합니다." }
```
사소한 내용이지만 check 의 조건과 메시지가 일치하는것이 더 읽기 쉽다고 생각합니다.